fusor-ember-cli/app/mixins/validated-input-mixin.js

```diff
@@ -53,9 +53,11 @@ export default Ember.Mixin.create({
     // this action is triggered on key-up. however, for password fields only, we don't want
     // to show validation errors after every key stroke unless 8 characters have been entered
     showValidationErrorsKeyUp() {
-      if (!(this.get('isPassword') && this.get('value.length') < 8)) {
-        this.set("showValidationError", true);
-      }
+      let self = this;
+      // delay showing validation for 1 second
+      Ember.run.later((function() {
+        self.set("showValidationError", true);
+      }), 2000);
```

Comment:
@cfchase, @eriknelson, I made a 2 sec rather than 1 sec delay. When testing, 1 sec seemed way too quick. Try it out and let me know.